src/screens/MenuScreen.tsx

```diff
@@ -80,24 +82,57 @@ export default function MenuScreen() {
 
 
     // Listen to when randRecipes get set
-    function onSwipedLeft(idx: number) {
+    async function onSwipedLeft(idx: number) {
         console.log('Swiped left');
+        const query = `query getUser {
+            users {
+              id
+              username
+            }
+          }`
+
+        const user = await axios({
+            url: 'https://savored-server.herokuapp.com/',
+            method: 'post',
+            data: {
+                query: query,
+            }
+        })
```

Comment:
We won't have to do this, we should be getting any user information from UserState

Reply:
This will eventually be an API call to the recipes tables